src/modules/core/components/InfoPopover/MemberInfoPopover/UserPermissions.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { ColonyRole } from '@colony/colony-js';
+import { defineMessages } from 'react-intl';
+
+import PermissionsLabel from '~core/PermissionsLabel';
+
+import styles from './MemberInfoPopover.css';
+import Heading from '~core/Heading';
+
+const displayName = `InfoPopover.MemberInfoPopover.UserPermissions`;
+
+interface Props {
+  roles: ColonyRole[];
+}
+
+const MSG = defineMessages({
+  labelText: {
+    id: 'InfoPopover.MemberInfoPopover.UserPermissions.labelText',
+    defaultMessage: 'Permissions',
+  },
+});
+
+const UserPermissions = ({ roles }: Props) => {
+  return (
+    <div className={styles.permissionsContainer}>
+      <Heading
+        appearance={{
+          size: 'normal',
+          margin: 'none',
+          theme: 'grey',
+          weight: 'bold',
+        }}
```

Comment:
So I know this has the `theme: grey` set, but for me it still renders as var `--dark`

![Screenshot from 2021-09-02 22-08-43](https://user-images.githubusercontent.com/1193222/131902661-b8057152-313a-4546-be89-1de38eda21f5.png)
![Screenshot from 2021-09-02 22-09-59](https://user-images.githubusercontent.com/1193222/131902663-02b33684-7218-4e09-afd4-70faad2c8f6f.png)

Please investigate, as this is the only thing that's not according to spec